app/models/user.rb

```diff
@@ -15,7 +15,7 @@ class User < ApplicationRecord
   def running_total
     @sum = 0
     self.requested_bookings.each do |booking|
-      @sum += booking.listing.currency_amount
+      @sum += booking.listing.currency_amount.to_i
```

Comment:
well spotted!

Reply:
it gave me an issue when I was looking through the app